### PR TITLE
Add code formatter

### PR DESCRIPTION
**Pull request by jingjingwang**

### CODE REVIEW

**Comment by jingjingwang** [comment]
The last commit will change once we have the pending PRs being merged, but the first two is the key so I think maybe it's okay for you to take a look even now.

File: build.gradle [hunk 1]
```diff
@@ -69,8 +78,26 @@ check.dependsOn testsuitesClasses
 /* Build the Eclipse .classpath file first, because if Gradle's build fails
  * we'll want to use Eclipse to debug it. */
 test.dependsOn eclipseClasspath
+/* Test if all *.java have been formatted. Run ./gradlew format to format all */
```
Comment (senderista) [inline build.gradle]: Shouldn't this comment move down to line 99?

**Comment by senderista** [comment]
LGTM

**Comment by jingjingwang** [comment]
Nice catch! (It used to be there)

**Pull request merged**